modules/eco-self-node-remediation-operator-about.adoc

```diff
@@ -0,0 +1,81 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/eco-self-node-remediation-operator.adoc
+
+:_content-type: CONCEPT
+[id="about-self-node-remediation-operator_{context}"]
+= About the Self Node Remediation Operator
+
+The Self Node Remediation Operator runs on the cluster nodes and reboots nodes that are identified as unhealthy. The Operator uses the `MachineHealthCheck` controller to detect the health of a node in the cluster. When a node is identified as unhealthy, the `MachineHealthCheck` resource creates the `SelfNodeRemediation` custom resource (CR), which triggers the Self Node Remediation Operator.
+
+The Self Node Remediation Operator minimizes downtime for stateful applications and restores compute capacity if transient failures occur. You can use this Operator regardless of the management interface, such as IPMI or an API to provision a node, and regardless of the cluster installation type, such as installer-provisioned infrastructure or user-provisioned infrastructure.
+
+[id="understanding-self-node-remediation-operator-config_{context}"]
+== Understanding the Self Node Remediation Operator configuration
+
+The Self Node Remediation Operator creates the `SelfNodeRemediationConfig` CR with the name `self-node-remediation-config` in the Self Node Remediation Operator's namespace. You can edit this CR. However, you cannot create a new CR for the Self Node Remediation Operator.
```

Comment:
`However, you cannot create a new CR for the Self Node Remediation Operator.`
- You can create (you even have such option in the UI), but this config will be ignored.

- You will see this messafe in the logs:
```
controllers.SelfNodeRemediationConfig
ignoring selfnoderemediationconfig CRs that are not named 'self-node-remediation-config'
or not in the namespace of the operator:
'openshift-operators' {"selfnoderemediationconfig":
"openshift-operators/selfnoderemediationconfig-copy"}

```

Reply:
Outside the scope of this PR, needs a bug